README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
+    <img src="assets/fiverr-mahiraani-20220811-logo/svg/DARVAZA_k.svg " width="100%">
```

Comment:
there is a space in `src`, and considering it's an SVG the desired size should be here I think